modules/unsupported/vector-mosaic/src/test/java/org/geotools/vectormosaic/VectorMosaicReaderTest.java

```diff
@@ -264,6 +283,9 @@ public void testLikeFilter() throws Exception {
         assertNotEquals(
                 Filter.INCLUDE,
                 ((VectorMosaicFeatureSource) featureSource).filterTracker.getGranuleFilter());
-        assertEquals(3, tracker.size()); // all indexes are hit, so all granules are checked
+        assertEquals(
+                3,
+                tracker.getGranuleNames()
+                        .size()); // all indexes are hit, so all granules are checked
```

Comment:
Play nice with the formatter. I've commented on this one, but there are other cases around like this.

```suggestion
        // all indexes are hit, so all granules are checked
        assertEquals(3,  tracker.getGranuleNames().size()); 
```

Reply:
fixed